bluepyopt/ephys/create_acc.py

```diff
@@ -0,0 +1,348 @@
+'''create a hoc file from a set of BluePyOpt.ephys parameters'''
+
+# pylint: disable=R0914
+
+import os
+
+from collections import namedtuple
+from glob import glob
+
+import numpy
+import jinja2
+
+from .create_hoc import Location, Range, _get_template_params
+
+
+# Define Neuron to Arbor variable conversions
+ArbVar = namedtuple('ArbVar', 'name, conv', 
+                    defaults=[None,None])
+
+_nrn2arb_var = dict(
+    cm=ArbVar(name='membrane-capacitance'), # conv=None implies identity
```

Comment:
`cm` -> `membrane-capacitance` does require a conversion factor. NEURON uses `uF/cm^2` while arbor uses `F/m^2`, so we would need to divide NEURON values by 100.

Reply:
Thanks for pointing this out - done in https://github.com/BlueBrain/BluePyOpt/pull/393/commits/d2f2f42e05e4a74caaaaa19d44453d42c5b0c947.